### DISCUSSION TITLE
Let user know when copilot has no suggestions

### DISCUSSION THREAD

**Discussion by snatchysquid** [community/community#7394] [Copilot Conversations]
Sometimes when writing the code, I end up waiting a long time for the copilot to complete my code, but nothing happens. It would be nice if the tool would let the user know when no such completion exists.

**Comment by KlausEverWalkingDev** [comment]
Great idea. Maybe a ellipsis or a emoticon like "):" and disabling the TAB.

**Comment by belle1306** [comment]
> Sometimes when writing the code, I end up waiting a long time for the copilot to complete my code, but nothing happens. It would be nice if the tool would let the user know when no such completion exists.
